### ISSUE TITLE
NAR hash mismatch in input

### ISSUE THREAD

**Issue by happysalada** [MatthewCroughan/nixinate#24]
here is the full log
```
🤞 Activating configuration non-hermetically on bee via ssh
+ /nix/store/85fcpkyv8qg4kpnv0p2nql2k7qp6q4ab-openssh-9.0p1/bin/ssh -t yt@24.48.87.104 'sudo nixos-rebuild switch --flake /nix/store/ln57g0jd3pqmdwfkb8vmca17hcfm4awv-source#bee'
[sudo] password for yt:
error: NAR hash mismatch in input 'github:nixos/nixpkgs/72d1b0d0fac131df1ea254b65413c85609bdd2ee' (/nix/store/3jvygw6vzsazkfhxcd66w9gcg7y91c3k-source), expected 'sha256-dLqquYHvj6/EYPljULXJVUKcizB/8cbgdIL7aEXTDZY=', got 'sha256-g5Wo75ddDQmWnL70rJCMm+JJlvHbzPFUePUpuMNn5qk='
(use '--show-trace' to show detailed location information)
```

just started getting this today, no idea why.
I'm executing from a darwin machine to build a nix machine.
I'm not using the hermetic feature.
Not sure where this is coming from.

**Comment by MatthewCroughan** [comment]
Is your full source code and flake that you're trying to work with available on Github anywhere?

**Comment by happysalada** [comment]
yes it is. I'm trying to build `bee` https://github.com/happysalada/dotfiles/blob/master/flake.nix#L68
by running `nix run .#apps.nixinate.bee`
you might just need to change the ip and ssh user and you should be fine.
(l think all files are checked by git).

**Comment by happysalada** [comment]
I thought originally that it was because the nix version was different on both machines and so those were computing the NAR differently. I verified since then that the nix version is the same on both machine.
I'm wondering if it would be a `darwin` vs `nixos` thing.
It feels like it has nothing to do with nixinate.
